app/src/main/java/com/glia/exampleapp/GliaWidgetsConfigManager.java

```diff
@@ -46,17 +46,21 @@ public static GliaWidgetsConfig obtainConfig(@NonNull Intent intent, @NonNull Co
     }
 
     @NonNull
-    private static GliaWidgetsConfig obtainConfigFromDeepLink(@NonNull Uri data, @NonNull Context applicationContext) {
+    public static GliaWidgetsConfig obtainConfigFromDeepLink(@NonNull Uri data, @NonNull Context applicationContext) {
         saveQueueIdToPrefs(data, applicationContext);
         saveVisitorContextAssetIdIfPresent(data, applicationContext);
+        saveSiteIdToPrefs(data, applicationContext);
 
 //        We're not checking availability for every query param separately because this is for acceptance tests and we assume that link would be correct
-        if (SECRET_KEY.equals(data.getLastPathSegment()))
+        if (SECRET_KEY.equals(data.getLastPathSegment())) {
+            saveSiteApiKeyAuthToPrefs(data, applicationContext);
             return obtainConfigFromSecretDeepLink(data, applicationContext);
-        else if (TOKEN_KEY.equals(data.getLastPathSegment()))
+        } else if (TOKEN_KEY.equals(data.getLastPathSegment())) {
+            saveAppTokenAuthToPrefs(data, applicationContext);
```

Comment:
NIT: As far as I understood we can remove the app token logic. But maube it can be done in separate PR